pkgs/by-name/pr/prism-model-checker-unwrapped/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  gccStdenv,
+  coreutils,
+  fetchFromGitHub,
+  openjdk,
+  cctools,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+let
+  stdenv' = stdenv;
```

Comment:
What? Why the double let-in?

Reply:
otherwise I get infinite recursion. I saw this patten  in another package, but I suppose in this case there is no reason to just use stdenv'